### DISCUSSION TITLE
gpt4free integration

### DISCUSSION THREAD

**Discussion by Felitendo** [open-webui/open-webui#5463] [General]
**Is your feature request related to a problem? Please describe.**
No

**Describe the solution you'd like**
Implement a way to use the API Endpoint of gpt4free

**Describe alternatives you've considered**
Using the gpt4free webui, but it's very bad

**Additional context**
https://github.com/xtekky/gpt4free

**Comment by cheahjs** [comment]
1. Not something we want to officially support because it's (mostly) reverse engineered APIs.
2. It already exposes an OpenAI-compatible endpoint.
